packages/mint-components/src/components/sqm-link-button/sqm-link-button.feature

```diff
@@ -0,0 +1,9 @@
+@author:noah
+@owner:noah
+
+Feature: Link Button
+
+	Scenario: When a user clicks the button a link open
+		When a user clicks the button
+		Then a link opens
```

Comment:
Spec incomplete

Reply:
bump on the above

Reply:
Added a second spec, not really much else to say here @DerekSiemens ?